components/form/form-label.component.ts

```diff
@@ -81,6 +85,17 @@ export class NzFormLabelComponent implements OnDestroy {
   }
   private _tooltipIcon: NzFormTooltipIcon | 'default' = 'default';
 
+  @Input()
+  set nzLabelWrap(value: boolean) {
```

Comment:
Can we use @InputBoolean() instead of getter?
```html
<nz-form-label nzLabelWrap></nz-form-label>
```

Reply:
Also you need to rebase this with the master branch.

Reply:
okay👌🏻

Reply:
> Can we use @InputBoolean() instead of getter?
> 
> ```
> <nz-form-label nzLabelWrap></nz-form-label>
> ```

If the nzLabelWrap of the NzFormLabelComponent has no value, we
 need to get the nzLabelWrap of the Form Directive,  so i think we need to use getter

Reply:
@InputBoolean() will automatically set this property to be true, check other components that use this decorator like `select`